### PR TITLE
Avoid selecting bitmap fonts from fontconfig

### PR DESCRIPTION
**Pull request by michaelwu**
I have a number of pcf fonts installed. One of them is Adobe Helvetica, and it turns out many websites try to use Helvetica. This crashes servo.

[Review on Reviewable](https://reviewable.io/reviews/servo/servo/6143)

### CODE REVIEW

File: components/gfx/platform/freetype/font_list.rs [hunk 1]
```diff
@@ -39,7 +40,19 @@ pub fn get_available_families<F>(mut callback: F) where F: FnMut(String) {
...
+            if FcPatternGetString(*font, FC_FONTFORMAT.as_ptr() as *mut c_char, v, &mut format) != FcResultMatch {
+                continue;
+            }
+
+            let fontformat = c_str_to_string(format as *const c_char);
+            if fontformat != "TrueType" &&
+               fontformat != "CFF" &&
+               fontformat != "Type 1" {
+                continue;
+            }
```
Comment (pcwalton) [inline components/gfx/platform/freetype/font_list.rs]: Could you add a comment over this block explaining what you're doing?

**Comment by michaelwu** [comment]
Done.

**Comment by pcwalton** [comment]
Looks good to me, squash away.

**Comment by michaelwu** [comment]
Squashed.

**Pull request merged**